py3status/modules/process_status.py

```diff
@@ -54,13 +60,14 @@ def _get_text(self):
     def process_status(self):
         if self.process is None:
             color = self.py3.COLOR_BAD
-            text = ERR_NO_PROCESS
+            text = self.py3.safe_format(self.format_error,
+                                        {'state': self.format_not_running})
         else:
             (color, text) = self._get_text()
 
         response = {
             'cached_until': self.py3.time_in(self.cache_timeout),
-            'full_text': text,
+            'full_text': self.py3.safe_format(self.format, {'state': text}),
```

Comment:
What is this for? process requires a binary name. None of that {binary} thing.

How to add / proceed from here?